lib/cuke_modeler/models/row.rb

```diff
@@ -25,6 +25,11 @@ def initialize(source_text = nil)
       populate_row(self, parsed_row_data)
     end
 
+    # Returns the model objects that belong to this model.
+    def children
+      cells
+    end
+
```

Comment:
I'll have to look around more later but, off hand, adding this method for just this class is suspicious. It feels like either functionality that already exists or a requirement that was somehow missed for only this class.

Reply:
Yep. That's a requirements hole that needs patching. I'll take care of it.